dvc/repo/experiments/__init__.py

```diff
@@ -623,3 +623,18 @@ def gc(self, *args, **kwargs):
         from dvc.repo.experiments.gc import gc
 
         return gc(self.repo, *args, **kwargs)
+
+    def push(self, *args, **kwargs):
+        from dvc.repo.experiments.push import push
+
+        return push(self.repo, *args, **kwargs)
+
+    def pull(self, *args, **kwargs):
+        from dvc.repo.experiments.pull import pull
+
+        return pull(self.repo, *args, **kwargs)
+
+    def list(self, *args, **kwargs):
+        from dvc.repo.experiments.list import list_
+
+        return list_(self.repo, *args, **kwargs)
```

Comment:
we have the same problem with Repo.ls :slightly_smiling_face: 
```suggestion
        from dvc.repo.experiments.ls import ls

        return ls(self.repo, *args, **kwargs)
```

Reply:
Maybe even Experiments.list -> Experiments.ls for the API. For CLI list works fine, same as in `dvc list`.